[Cloud Security] Allow open Misconfiguration Flyout from Contextual Flyout

animehart opened the pull request:
## Summary

Users now are able to open Findings Flyout via Contextual Flyout

https://github.com/user-attachments/assets/cf7d5382-aebd-4041-99f3-ce982ae317e4

animehart committed:
trial run

animehart committed:
trying

animehart committed:
cleaning up a bit, todo: fix json tab css issue

animehart committed:
fix i18n

animehart committed:
fix ftr

opauloh committed:
Merge branch 'main' into csp-update-misconfiguration-flyout-ui

animehart committed:
truncate long resource name

animehart committed:
Merge branch 'csp-update-misconfiguration-flyout-ui' of github.com:animehart/kibana into csp-update-misconfiguration-flyout-ui

animehart committed:
fix json

animehart committed:
fix check types error

animehart committed:
initial

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-cloud-from-pr --> Click to trigger **kibana-deploy-cloud-from-pr** for this PR!

animehart committed:
more updates on preview

animehart committed:
fix merge

animehart commented:
/ci

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/292985)
* Commit: 36c00ea54ee394cc0ef87b88d207682519bcd326

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/292985#0196370c-f6c1-41a8-b655-d65e38f6aea3)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292985","number":292985,"commit":"36c00ea54ee394cc0ef87b88d207682519bcd326"}],"number":292985}
buildkite-pr-comment-->

animehart committed:
fix quick checks

animehart commented:
/ci

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/292989)
* Commit: 00fd202ba25fa077d551b3d80c1efcf8d741c6e3

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/292989#01963747-cfb0-49f6-9fd2-66db08f8c267)

### History
* :broken_heart: [Build #292985](https://buildkite.com/elastic/kibana-pull-request/builds/292985) failed 36c00ea54ee394cc0ef87b88d207682519bcd326

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292989","number":292989,"commit":"00fd202ba25fa077d551b3d80c1efcf8d741c6e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292985","number":292985,"commit":"36c00ea54ee394cc0ef87b88d207682519bcd326"}],"number":292989}
buildkite-pr-comment-->

animehart committed:
fix check again

animehart commented:
/ci

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/292994)
* Commit: c85f02563af95ae7db33736a89d1f0abf1500a2b

### Failed CI Steps
* [FTR Configs #124](https://buildkite.com/elastic/kibana-pull-request/builds/292994#019637b6-6cbb-40fb-982f-f4975e22bb5d)
* [FTR Configs #124](https://buildkite.com/elastic/kibana-pull-request/builds/292994#019637de-6623-45a8-b383-9e4ab2ef0300)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/292994#019637b6-6cbb-40fb-982f-f4975e22bb5d) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/292994/jobs/019637b6-6cbb-40fb-982f-f4975e22bb5d/artifacts/019637de-3432-43f5-b7b7-80a675d4f7de) FTR Configs #<span></span>124 / console app console settings displays the a11y overlay
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/292994#019637de-6623-45a8-b383-9e4ab2ef0300) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/292994/jobs/019637de-6623-45a8-b383-9e4ab2ef0300/artifacts/019637e4-e242-464d-a24b-8f88b73461ca) FTR Configs #<span></span>124 / console app console settings displays the a11y overlay

### History
* :broken_heart: [Build #292989](https://buildkite.com/elastic/kibana-pull-request/builds/292989) failed 00fd202ba25fa077d551b3d80c1efcf8d741c6e3
* :broken_heart: [Build #292985](https://buildkite.com/elastic/kibana-pull-request/builds/292985) failed 36c00ea54ee394cc0ef87b88d207682519bcd326

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292994","number":292994,"commit":"c85f02563af95ae7db33736a89d1f0abf1500a2b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292989","number":292989,"commit":"00fd202ba25fa077d551b3d80c1efcf8d741c6e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292985","number":292985,"commit":"36c00ea54ee394cc0ef87b88d207682519bcd326"}],"number":292994}
buildkite-pr-comment-->

animehart committed:
removing preview version as per discussion

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293869)
* Commit: 875e7fbd106d883bae96165fcc3a8e92a301831b

### Failed CI Steps
* [Serverless Investigations - Security Solution Cypress Tests #9](https://buildkite.com/elastic/kibana-pull-request/builds/293869#01963f90-a21e-4d7b-9f95-42e3982b86ff)

### History
* :broken_heart: [Build #292994](https://buildkite.com/elastic/kibana-pull-request/builds/292994) failed c85f02563af95ae7db33736a89d1f0abf1500a2b
* :broken_heart: [Build #292989](https://buildkite.com/elastic/kibana-pull-request/builds/292989) failed 00fd202ba25fa077d551b3d80c1efcf8d741c6e3
* :broken_heart: [Build #292985](https://buildkite.com/elastic/kibana-pull-request/builds/292985) failed 36c00ea54ee394cc0ef87b88d207682519bcd326

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293869","number":293869,"commit":"875e7fbd106d883bae96165fcc3a8e92a301831b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292994","number":292994,"commit":"c85f02563af95ae7db33736a89d1f0abf1500a2b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292989","number":292989,"commit":"00fd202ba25fa077d551b3d80c1efcf8d741c6e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292985","number":292985,"commit":"36c00ea54ee394cc0ef87b88d207682519bcd326"}],"number":293869}
buildkite-pr-comment-->

animehart committed:
removing scopeid for now, will re add it later when we actually need it

animehart committed:
Merge branch 'main' into csp-update-misconfiguration-flyout-ui-next-part

kibanamachine committed:
[CI] Auto-commit changed files from 'node scripts/eslint --no-cache --fix'

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/293905)
* Commit: 7f19f1d50c960fb04bf81e1eab94da46b7cc6c88

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/31f0f211d9d76fc757adaff91e41f94bafbacfb4) | [after](https://github.com/elastic/kibana/commit/7f19f1d50c960fb04bf81e1eab94da46b7cc6c88) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 524.3KB | 524.3KB | +37.0B |
| `securitySolution` | 9.0MB | 9.0MB | -110.0B |
| total |  |  | -73.0B |



### History
* :broken_heart: [Build #293869](https://buildkite.com/elastic/kibana-pull-request/builds/293869) failed 875e7fbd106d883bae96165fcc3a8e92a301831b
* :broken_heart: [Build #292994](https://buildkite.com/elastic/kibana-pull-request/builds/292994) failed c85f02563af95ae7db33736a89d1f0abf1500a2b
* :broken_heart: [Build #292989](https://buildkite.com/elastic/kibana-pull-request/builds/292989) failed 00fd202ba25fa077d551b3d80c1efcf8d741c6e3
* :broken_heart: [Build #292985](https://buildkite.com/elastic/kibana-pull-request/builds/292985) failed 36c00ea54ee394cc0ef87b88d207682519bcd326

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293905","number":293905,"commit":"7f19f1d50c960fb04bf81e1eab94da46b7cc6c88"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293869","number":293869,"commit":"875e7fbd106d883bae96165fcc3a8e92a301831b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292994","number":292994,"commit":"c85f02563af95ae7db33736a89d1f0abf1500a2b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292989","number":292989,"commit":"00fd202ba25fa077d551b3d80c1efcf8d741c6e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292985","number":292985,"commit":"36c00ea54ee394cc0ef87b88d207682519bcd326"}],"number":293905}
buildkite-pr-comment-->

seanrathier commented on x-pack/solutions/security/plugins/cloud_security_posture/public/common/hooks/use_expandable_flyout_csp.ts:
```diff
@@ -10,7 +10,9 @@ import { CspFinding } from '@kbn/cloud-security-posture-common';
 
-export const useExpandableFlyoutCsp = () => {
+export const useExpandableFlyoutCsp = (
+  flyoutType: 'misconfiguration' | 'vulnerability' = 'misconfiguration'
```
Am I missing something here?  We added the flyoutType but there is nothing using it? 😅

animehart replied:
oh woops 
this was supposed to go on my next PR 😄 (when we have Vulnerability Flyout as well)
I'll remove it for now

seanrathier reviewed (commented):
Look good overall, just one concern

animehart committed:
pr comments + added copy function on table, todo: fix truncate text

animehart committed:
added divider between accordions

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294373)
* Commit: 3c8bf01a3dafc1d5d3470f6915dbe05b4a691938

### Failed CI Steps
* [Defend Workflows Cypress Tests #2](https://buildkite.com/elastic/kibana-pull-request/builds/294373#019647a8-fdb7-4ea1-955b-36b751d1b839)

### History
* :green_heart: [Build #293905](https://buildkite.com/elastic/kibana-pull-request/builds/293905) succeeded 7f19f1d50c960fb04bf81e1eab94da46b7cc6c88
* :broken_heart: [Build #293869](https://buildkite.com/elastic/kibana-pull-request/builds/293869) failed 875e7fbd106d883bae96165fcc3a8e92a301831b
* :broken_heart: [Build #292994](https://buildkite.com/elastic/kibana-pull-request/builds/292994) failed c85f02563af95ae7db33736a89d1f0abf1500a2b
* :broken_heart: [Build #292989](https://buildkite.com/elastic/kibana-pull-request/builds/292989) failed 00fd202ba25fa077d551b3d80c1efcf8d741c6e3
* :broken_heart: [Build #292985](https://buildkite.com/elastic/kibana-pull-request/builds/292985) failed 36c00ea54ee394cc0ef87b88d207682519bcd326

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294373","number":294373,"commit":"3c8bf01a3dafc1d5d3470f6915dbe05b4a691938"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293905","number":293905,"commit":"7f19f1d50c960fb04bf81e1eab94da46b7cc6c88"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293869","number":293869,"commit":"875e7fbd106d883bae96165fcc3a8e92a301831b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292994","number":292994,"commit":"c85f02563af95ae7db33736a89d1f0abf1500a2b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292989","number":292989,"commit":"00fd202ba25fa077d551b3d80c1efcf8d741c6e3"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292985","number":292985,"commit":"36c00ea54ee394cc0ef87b88d207682519bcd326"}],"number":294373}
buildkite-pr-comment-->

animehart committed:
ui update

animehart committed:
Merge branch 'main' into csp-update-misconfiguration-flyout-ui-next-part

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/294726)
* Commit: c30ceaadb2461ef6b1e73ca29f39509b7ee16eec

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/9a66ec9348ad729839a7991fc16ae33cb35fdd24) | [after](https://github.com/elastic/kibana/commit/c30ceaadb2461ef6b1e73ca29f39509b7ee16eec) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 521.6KB | 522.3KB | +730.0B |
| `securitySolution` | 9.0MB | 9.0MB | -110.0B |
| total |  |  | +620.0B |



### History
* :broken_heart: [Build #294373](https://buildkite.com/elastic/kibana-pull-request/builds/294373) failed 3c8bf01a3dafc1d5d3470f6915dbe05b4a691938
* :green_heart: [Build #293905](https://buildkite.com/elastic/kibana-pull-request/builds/293905) succeeded 7f19f1d50c960fb04bf81e1eab94da46b7cc6c88
* :broken_heart: [Build #293869](https://buildkite.com/elastic/kibana-pull-request/builds/293869) failed 875e7fbd106d883bae96165fcc3a8e92a301831b
* :broken_heart: [Build #292994](https://buildkite.com/elastic/kibana-pull-request/builds/292994) failed c85f02563af95ae7db33736a89d1f0abf1500a2b
* :broken_heart: [Build #292989](https://buildkite.com/elastic/kibana-pull-request/builds/292989) failed 00fd202ba25fa077d551b3d80c1efcf8d741c6e3

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294726","number":294726,"commit":"c30ceaadb2461ef6b1e73ca29f39509b7ee16eec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294373","number":294373,"commit":"3c8bf01a3dafc1d5d3470f6915dbe05b4a691938"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293905","number":293905,"commit":"7f19f1d50c960fb04bf81e1eab94da46b7cc6c88"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293869","number":293869,"commit":"875e7fbd106d883bae96165fcc3a8e92a301831b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292994","number":292994,"commit":"c85f02563af95ae7db33736a89d1f0abf1500a2b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/292989","number":292989,"commit":"00fd202ba25fa077d551b3d80c1efcf8d741c6e3"}],"number":294726}
buildkite-pr-comment-->

seanrathier commented on x-pack/solutions/security/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_right/header.tsx:
```diff
@@ -64,13 +64,26 @@ export const FindingsMisconfigurationFlyoutHeader = ({
-                          </EuiToolTip>
-                        </EuiFlexItem>
-                        <EuiFlexItem>
+                      <EuiToolTip content={resourceName} position="top">
```
We are duplicating this component in the `overview_tab.tsx`.  We should create a common component.

seanrathier reviewed (commented):
Looks great,  one comment to discuss.

animehart committed:
pr comments

animehart committed:
Merge branch 'csp-update-misconfiguration-flyout-ui-next-part' of github.com:animehart/kibana into csp-update-misconfiguration-flyout-ui-next-part

animehart committed:
Merge branch 'main' into csp-update-misconfiguration-flyout-ui-next-part

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/295144)
* Commit: 5174370f021f46e80c9c12032f9cf2b8a2bb4c3e

### Failed CI Steps
* [Defend Workflows Cypress Tests #5](https://buildkite.com/elastic/kibana-pull-request/builds/295144#01965f5a-90f6-4a03-98d6-88a2a77ba3a6)

### History
* :broken_heart: [Build #295102](https://buildkite.com/elastic/kibana-pull-request/builds/295102) failed 212f869adfe75680c3f8a1f7a06c213dffc65bcf
* :green_heart: [Build #294726](https://buildkite.com/elastic/kibana-pull-request/builds/294726) succeeded c30ceaadb2461ef6b1e73ca29f39509b7ee16eec
* :broken_heart: [Build #294373](https://buildkite.com/elastic/kibana-pull-request/builds/294373) failed 3c8bf01a3dafc1d5d3470f6915dbe05b4a691938
* :green_heart: [Build #293905](https://buildkite.com/elastic/kibana-pull-request/builds/293905) succeeded 7f19f1d50c960fb04bf81e1eab94da46b7cc6c88
* :broken_heart: [Build #293869](https://buildkite.com/elastic/kibana-pull-request/builds/293869) failed 875e7fbd106d883bae96165fcc3a8e92a301831b

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295144","number":295144,"commit":"5174370f021f46e80c9c12032f9cf2b8a2bb4c3e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295102","number":295102,"commit":"212f869adfe75680c3f8a1f7a06c213dffc65bcf"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294726","number":294726,"commit":"c30ceaadb2461ef6b1e73ca29f39509b7ee16eec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294373","number":294373,"commit":"3c8bf01a3dafc1d5d3470f6915dbe05b4a691938"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293905","number":293905,"commit":"7f19f1d50c960fb04bf81e1eab94da46b7cc6c88"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293869","number":293869,"commit":"875e7fbd106d883bae96165fcc3a8e92a301831b"}],"number":295144}
buildkite-pr-comment-->

animehart committed:
Merge branch 'main' into csp-update-misconfiguration-flyout-ui-next-part

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/295168)
* Commit: 005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 522.0KB | 522.3KB | +299.0B |
| `securitySolution` | 9.0MB | 9.0MB | -110.0B |
| total |  |  | +189.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 17.4KB | 17.4KB | +18.0B |



### History
* :broken_heart: [Build #295144](https://buildkite.com/elastic/kibana-pull-request/builds/295144) failed 5174370f021f46e80c9c12032f9cf2b8a2bb4c3e
* :broken_heart: [Build #295102](https://buildkite.com/elastic/kibana-pull-request/builds/295102) failed 212f869adfe75680c3f8a1f7a06c213dffc65bcf
* :green_heart: [Build #294726](https://buildkite.com/elastic/kibana-pull-request/builds/294726) succeeded c30ceaadb2461ef6b1e73ca29f39509b7ee16eec
* :broken_heart: [Build #294373](https://buildkite.com/elastic/kibana-pull-request/builds/294373) failed 3c8bf01a3dafc1d5d3470f6915dbe05b4a691938
* :green_heart: [Build #293905](https://buildkite.com/elastic/kibana-pull-request/builds/293905) succeeded 7f19f1d50c960fb04bf81e1eab94da46b7cc6c88

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295168","number":295168,"commit":"005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295144","number":295144,"commit":"5174370f021f46e80c9c12032f9cf2b8a2bb4c3e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295102","number":295102,"commit":"212f869adfe75680c3f8a1f7a06c213dffc65bcf"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294726","number":294726,"commit":"c30ceaadb2461ef6b1e73ca29f39509b7ee16eec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294373","number":294373,"commit":"3c8bf01a3dafc1d5d3470f6915dbe05b4a691938"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/293905","number":293905,"commit":"7f19f1d50c960fb04bf81e1eab94da46b7cc6c88"}],"number":295168}
buildkite-pr-comment-->

animehart commented:
Holding Merge until more clarification from Product regarding the new behavior

opauloh reviewed (changes requested):
Requesting changes for converting the `CopyableText` to a component instead of a function in order to be more reusable.

animehart committed:
addressing CopyableText Component commment

animehart committed:
Merge branch 'csp-update-misconfiguration-flyout-ui-next-part' of github.com:animehart/kibana into csp-update-misconfiguration-flyout-ui-next-part

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/295615)
* Commit: bafbd941f160949d8936d3d560dcf968ac6f0601

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/bafbd941f160949d8936d3d560dcf968ac6f0601) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 522.0KB | 522.4KB | +374.0B |
| `securitySolution` | 9.0MB | 9.0MB | -110.0B |
| total |  |  | +264.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/bafbd941f160949d8936d3d560dcf968ac6f0601) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 17.4KB | 17.4KB | +18.0B |



### History
* :green_heart: [Build #295168](https://buildkite.com/elastic/kibana-pull-request/builds/295168) succeeded 005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0
* :broken_heart: [Build #295144](https://buildkite.com/elastic/kibana-pull-request/builds/295144) failed 5174370f021f46e80c9c12032f9cf2b8a2bb4c3e
* :broken_heart: [Build #295102](https://buildkite.com/elastic/kibana-pull-request/builds/295102) failed 212f869adfe75680c3f8a1f7a06c213dffc65bcf
* :green_heart: [Build #294726](https://buildkite.com/elastic/kibana-pull-request/builds/294726) succeeded c30ceaadb2461ef6b1e73ca29f39509b7ee16eec
* :broken_heart: [Build #294373](https://buildkite.com/elastic/kibana-pull-request/builds/294373) failed 3c8bf01a3dafc1d5d3470f6915dbe05b4a691938
* :green_heart: [Build #293905](https://buildkite.com/elastic/kibana-pull-request/builds/293905) succeeded 7f19f1d50c960fb04bf81e1eab94da46b7cc6c88

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295615","number":295615,"commit":"bafbd941f160949d8936d3d560dcf968ac6f0601"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295168","number":295168,"commit":"005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295144","number":295144,"commit":"5174370f021f46e80c9c12032f9cf2b8a2bb4c3e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295102","number":295102,"commit":"212f869adfe75680c3f8a1f7a06c213dffc65bcf"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294726","number":294726,"commit":"c30ceaadb2461ef6b1e73ca29f39509b7ee16eec"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294373","number":294373,"commit":"3c8bf01a3dafc1d5d3470f6915dbe05b4a691938"}],"number":295615}
buildkite-pr-comment-->

opauloh commented on x-pack/solutions/security/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/overview_tab.tsx:
```diff
@@ -48,7 +50,12 @@ const columns: Array<EuiBasicTableColumn<any>> = [
     name: 'Value',
-    truncateText: true,
+    truncateText: false,
+    render: (value: string) => (
+      <>
+        <TruncatedCopyableText textToCopy={value} />
+      </>
+    ),
```
nit: 
```suggestion
    render: (value: string) => (<TruncatedCopyableText textToCopy={value} />),
```

opauloh reviewed (approved):
LGTM, thanks for addressing my comments

animehart committed:
added preview mode + behaviour update

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/295932)
* Commit: 3e0733d6fec9b3b65ae45c46e39669e4e1fb314a

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/3e0733d6fec9b3b65ae45c46e39669e4e1fb314a) | diff |
| --- | --- | --- | --- |
| `@kbn/cloud-security-posture` | 130 | 134 | +4 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/3e0733d6fec9b3b65ae45c46e39669e4e1fb314a) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 522.0KB | 522.4KB | +374.0B |
| `securitySolution` | 9.0MB | 9.0MB | +837.0B |
| total |  |  | +1.2KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/3e0733d6fec9b3b65ae45c46e39669e4e1fb314a) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 17.4KB | 17.4KB | +18.0B |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/2df2e7890698677b617660a0919a3d277add6fb4) | [after](https://github.com/elastic/kibana/commit/3e0733d6fec9b3b65ae45c46e39669e4e1fb314a) | diff |
| --- | --- | --- | --- |
| `@kbn/cloud-security-posture` | 131 | 135 | +4 |


</details>


### History
* :green_heart: [Build #295615](https://buildkite.com/elastic/kibana-pull-request/builds/295615) succeeded bafbd941f160949d8936d3d560dcf968ac6f0601
* :green_heart: [Build #295168](https://buildkite.com/elastic/kibana-pull-request/builds/295168) succeeded 005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0
* :broken_heart: [Build #295144](https://buildkite.com/elastic/kibana-pull-request/builds/295144) failed 5174370f021f46e80c9c12032f9cf2b8a2bb4c3e
* :broken_heart: [Build #295102](https://buildkite.com/elastic/kibana-pull-request/builds/295102) failed 212f869adfe75680c3f8a1f7a06c213dffc65bcf
* :green_heart: [Build #294726](https://buildkite.com/elastic/kibana-pull-request/builds/294726) succeeded c30ceaadb2461ef6b1e73ca29f39509b7ee16eec
* :broken_heart: [Build #294373](https://buildkite.com/elastic/kibana-pull-request/builds/294373) failed 3c8bf01a3dafc1d5d3470f6915dbe05b4a691938

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295932","number":295932,"commit":"3e0733d6fec9b3b65ae45c46e39669e4e1fb314a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295615","number":295615,"commit":"bafbd941f160949d8936d3d560dcf968ac6f0601"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295168","number":295168,"commit":"005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295144","number":295144,"commit":"5174370f021f46e80c9c12032f9cf2b8a2bb4c3e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295102","number":295102,"commit":"212f869adfe75680c3f8a1f7a06c213dffc65bcf"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/294726","number":294726,"commit":"c30ceaadb2461ef6b1e73ca29f39509b7ee16eec"}],"number":295932}
buildkite-pr-comment-->

opauloh reviewed (changes requested):
I believe we need to address the type changes in order to prevent unexpected issues at runtime

opauloh committed:
Refactors CSP flyout props for preview mode

Updates the CSP misconfiguration flyout to differentiate between preview and non-preview modes using distinct prop types.

This change improves type safety and provides better context for the flyout component.

animehart committed:
Merge pull request #4 from opauloh/misconfiguration-types-improvement

Refactors CSP flyout props for preview mode

animehart committed:
Merge branch 'main' into csp-update-misconfiguration-flyout-ui-next-part

animehart commented on x-pack/solutions/security/plugins/security_solution/public/flyout/csp_details/findings_flyout/findings_right/index.tsx:
```diff
@@ -57,18 +79,42 @@ export const FindingsMisconfigurationPanel = ({
-              <EuiFlyoutFooter>
-                <CspFlyout.Footer createRuleFn={createRuleFn} />
-              </EuiFlyoutFooter>
+              {!isPreviewMode ? (
```
its part of the decision we made, so for preview mode we will hide the Footer for the Flyout.

opauloh replied:
got it, thanks

animehart committed:
minor fix

opauloh reviewed (approved):
🚀

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/296451)
* Commit: 63ee9283fffaec3929a9b469bde0a99ee386f281

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/f6dbd0df5856f728d3d38c3827444d89b6088984) | [after](https://github.com/elastic/kibana/commit/63ee9283fffaec3929a9b469bde0a99ee386f281) | diff |
| --- | --- | --- | --- |
| `@kbn/cloud-security-posture` | 130 | 129 | -1 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/f6dbd0df5856f728d3d38c3827444d89b6088984) | [after](https://github.com/elastic/kibana/commit/63ee9283fffaec3929a9b469bde0a99ee386f281) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 516.8KB | 517.1KB | +374.0B |
| `securitySolution` | 9.1MB | 9.1MB | +790.0B |
| total |  |  | +1.1KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/f6dbd0df5856f728d3d38c3827444d89b6088984) | [after](https://github.com/elastic/kibana/commit/63ee9283fffaec3929a9b469bde0a99ee386f281) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 18.0KB | 18.0KB | +18.0B |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/f6dbd0df5856f728d3d38c3827444d89b6088984) | [after](https://github.com/elastic/kibana/commit/63ee9283fffaec3929a9b469bde0a99ee386f281) | diff |
| --- | --- | --- | --- |
| `@kbn/cloud-security-posture` | 131 | 130 | -1 |


</details>


### History
* :green_heart: [Build #295932](https://buildkite.com/elastic/kibana-pull-request/builds/295932) succeeded 3e0733d6fec9b3b65ae45c46e39669e4e1fb314a
* :green_heart: [Build #295615](https://buildkite.com/elastic/kibana-pull-request/builds/295615) succeeded bafbd941f160949d8936d3d560dcf968ac6f0601
* :green_heart: [Build #295168](https://buildkite.com/elastic/kibana-pull-request/builds/295168) succeeded 005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0
* :broken_heart: [Build #295144](https://buildkite.com/elastic/kibana-pull-request/builds/295144) failed 5174370f021f46e80c9c12032f9cf2b8a2bb4c3e
* :broken_heart: [Build #295102](https://buildkite.com/elastic/kibana-pull-request/builds/295102) failed 212f869adfe75680c3f8a1f7a06c213dffc65bcf
* :green_heart: [Build #294726](https://buildkite.com/elastic/kibana-pull-request/builds/294726) succeeded c30ceaadb2461ef6b1e73ca29f39509b7ee16eec

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/296451","number":296451,"commit":"63ee9283fffaec3929a9b469bde0a99ee386f281"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295932","number":295932,"commit":"3e0733d6fec9b3b65ae45c46e39669e4e1fb314a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295615","number":295615,"commit":"bafbd941f160949d8936d3d560dcf968ac6f0601"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295168","number":295168,"commit":"005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295144","number":295144,"commit":"5174370f021f46e80c9c12032f9cf2b8a2bb4c3e"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295102","number":295102,"commit":"212f869adfe75680c3f8a1f7a06c213dffc65bcf"}],"number":296451}
buildkite-pr-comment-->

abhishekbhatia1710 reviewed (approved):
LGTM from entity analytics side

animehart committed:
Merge branch 'main' into csp-update-misconfiguration-flyout-ui-next-part

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/296796)
* Commit: 977acba5b7559a5ea9cf0a8841f062985369b850

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/8e1a426c25497f332fd516dee46061c900618e96) | [after](https://github.com/elastic/kibana/commit/977acba5b7559a5ea9cf0a8841f062985369b850) | diff |
| --- | --- | --- | --- |
| `@kbn/cloud-security-posture` | 130 | 129 | -1 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/8e1a426c25497f332fd516dee46061c900618e96) | [after](https://github.com/elastic/kibana/commit/977acba5b7559a5ea9cf0a8841f062985369b850) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 516.8KB | 517.1KB | +374.0B |
| `securitySolution` | 9.1MB | 9.1MB | +790.0B |
| total |  |  | +1.1KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/8e1a426c25497f332fd516dee46061c900618e96) | [after](https://github.com/elastic/kibana/commit/977acba5b7559a5ea9cf0a8841f062985369b850) | diff |
| --- | --- | --- | --- |
| `cloudSecurityPosture` | 18.0KB | 18.0KB | +18.0B |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/8e1a426c25497f332fd516dee46061c900618e96) | [after](https://github.com/elastic/kibana/commit/977acba5b7559a5ea9cf0a8841f062985369b850) | diff |
| --- | --- | --- | --- |
| `@kbn/cloud-security-posture` | 131 | 130 | -1 |


</details>


### History
* :green_heart: [Build #296451](https://buildkite.com/elastic/kibana-pull-request/builds/296451) succeeded 63ee9283fffaec3929a9b469bde0a99ee386f281
* :green_heart: [Build #295932](https://buildkite.com/elastic/kibana-pull-request/builds/295932) succeeded 3e0733d6fec9b3b65ae45c46e39669e4e1fb314a
* :green_heart: [Build #295615](https://buildkite.com/elastic/kibana-pull-request/builds/295615) succeeded bafbd941f160949d8936d3d560dcf968ac6f0601
* :green_heart: [Build #295168](https://buildkite.com/elastic/kibana-pull-request/builds/295168) succeeded 005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0
* :broken_heart: [Build #295144](https://buildkite.com/elastic/kibana-pull-request/builds/295144) failed 5174370f021f46e80c9c12032f9cf2b8a2bb4c3e
* :broken_heart: [Build #295102](https://buildkite.com/elastic/kibana-pull-request/builds/295102) failed 212f869adfe75680c3f8a1f7a06c213dffc65bcf

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/296796","number":296796,"commit":"977acba5b7559a5ea9cf0a8841f062985369b850"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/296451","number":296451,"commit":"63ee9283fffaec3929a9b469bde0a99ee386f281"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295932","number":295932,"commit":"3e0733d6fec9b3b65ae45c46e39669e4e1fb314a"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295615","number":295615,"commit":"bafbd941f160949d8936d3d560dcf968ac6f0601"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295168","number":295168,"commit":"005fa3510c5b1d1b1f33dee40d4c14b98f95c1d0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/295144","number":295144,"commit":"5174370f021f46e80c9c12032f9cf2b8a2bb4c3e"}],"number":296796}
buildkite-pr-comment-->
